diff --git a/node_modules/react-native-size-matters/lib/scaling-utils.js b/node_modules/react-native-size-matters/lib/scaling-utils.js
index e89372a..52413d0 100644
--- a/node_modules/react-native-size-matters/lib/scaling-utils.js
+++ b/node_modules/react-native-size-matters/lib/scaling-utils.js
@@ -3,16 +3,11 @@ import { Dimensions } from 'react-native';
 const { width, height } = Dimensions.get('window');
 const [shortDimension, longDimension] = width < height ? [width, height] : [height, width];
 
-//Default guideline sizes are based on standard ~5" screen mobile device
-const guidelineBaseWidth = 350;
-const guidelineBaseHeight = 680;
+const guidelineBaseWidth = 375;
+const guidelineBaseHeight = 812;
 
-export const scale = size => shortDimension / guidelineBaseWidth * size;
-export const verticalScale = size => longDimension / guidelineBaseHeight * size;
-export const moderateScale = (size, factor = 0.5) => size + (scale(size) - size) * factor;
-export const moderateVerticalScale = (size, factor = 0.5) => size + (verticalScale(size) - size) * factor;
+const scale = (size) => (shortDimension / guidelineBaseWidth) * size;
+const verticalScale = (size) => (longDimension / guidelineBaseHeight) * size;
+const moderateScale = (size, factor = 0.5) => size + (scale(size) - size) * factor;
 
-export const s = scale;
-export const vs = verticalScale;
-export const ms = moderateScale;
-export const mvs = moderateVerticalScale;
\ No newline at end of file
+export { scale, verticalScale, moderateScale };
